.github/workflows/deploy.yml

```diff
@@ -133,8 +133,14 @@ jobs:
       - name: Upload package to S3
         working-directory: ${{ env.package_path }}
         run: |
-          aws s3 rm "s3://${{ env.s3bucket }}/${{ env.s3path }}/" --recursive --exclude '*' --include '*.exe'
+          Write-Output "# S3 listing"
+          aws s3 ls "s3://${{ env.s3bucket }}/${{ env.s3path }}/" --recursive --human-readable
+          Write-Output "# S3 remove '*-$("${{ matrix.cmake_generator_platform }}".Replace("x64", "win64"))_*.exe'"
+          aws s3 rm "s3://${{ env.s3bucket }}/${{ env.s3path }}/" --recursive --exclude '*' --include "*-$("${{ matrix.cmake_generator_platform }}".Replace("x64", "win64"))_*.exe"
+          aws s3 ls "s3://${{ env.s3bucket }}/${{ env.s3path }}/" --recursive --human-readable
+          Write-Output "# S3 copy from '${{ env.package_path }}'"
           aws s3 cp . "s3://${{ env.s3bucket }}/${{ env.s3path }}/" --recursive
+          aws s3 ls "s3://${{ env.s3bucket }}/${{ env.s3path }}/" --recursive --human-readable
```

Comment:
The 3 `ls` calls are redundant here. It might make sense to  keep the first one to see what files are currently present, but the `rm` and `cp` commands already provide logs as to what files are deleted/copied.

See https://github.com/openDAQ/openDAQ/actions/runs/7626615223/job/20773393943 "Upload package to S3"

Reply:
removed